samplers/jaegerremote/sampler_remote_options.go

```diff
@@ -0,0 +1,145 @@
+// Copyright The OpenTelemetry Authors
+// Copyright (c) 2021 The Jaeger Authors.
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jaegerremote
+
+import (
+	"time"
+
+	"go.opentelemetry.io/otel/sdk/trace"
+)
+
+// SamplerOption is a function that sets some option on the sampler
+type SamplerOption func(options *samplerConfig)
+
+type samplerConfig struct {
+	sampler                 trace.Sampler
+	samplingServerURL       string
+	samplingRefreshInterval time.Duration
+	samplingFetcher         samplingStrategyFetcher
+	samplingParser          samplingStrategyParser
+	updaters                []samplerUpdater
+	posParams               perOperationSamplerParams
+}
+
+type config struct {
+	c samplerConfig
+}
```

Comment:
```suggestion
type config struct {
	sampler                 trace.Sampler
	samplingServerURL       string
	samplingRefreshInterval time.Duration
	samplingFetcher         samplingStrategyFetcher
	samplingParser          samplingStrategyParser
	updaters                []samplerUpdater
	posParams               perOperationSamplerParams
}
```

Extrapolate this to the functions that operate on `config`.

Reply:
thx